ui/templates/dialogs/upload-asset-dialog.html

```diff
@@ -17,6 +17,12 @@
             }
         });
 
+        let title = "Upload assets";
+        if (assetName){
+            title = "Upload into asset " + assetName;
```

Comment:
You can format the asset name as a badge or a bold element to highlight the information a bit more